log.go

```diff
@@ -1,12 +1,6 @@
 package breaker
 
 type Logger interface {
-	Info(v ...interface{})
-	Infof(format string, v ...interface{})
-	Warn(v ...interface{})
-	Warnf(format string, v ...interface{})
-	Error(v ...interface{})
-	Errorf(format string, v ...interface{})
-	Fatal(v ...interface{})
-	Fatalf(format string, v ...interface{})
+	Warn(msg string, args ...interface{})
+	Error(msg string, args ...interface{})
```

Comment:
This is a breaking API change.

However, since this pkg is at [v0.1.2](https://github.com/goware/breaker/releases/tag/v0.1.2), I think it's fine to accept this change and release v0.2.0.

@david-littlefarmer As long as you volunteer to update breaker everywhere? :D

Reply:
Yes it is breaking change, but that is not an issue, since this is not used much and we want to use `log/slog` everywhere anyways.